src/components/Card.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Card from './Card';
+import { mount, shallow } from 'enzyme';
+
+describe('Card', () => {
+  test('onClick handler working', () => {
+    // const textValue =  'Encouraging message';
+    const onClick = jest.fn();
+    const wrapper = shallow(
+      <button type="submit" onClick={onClick}></button>
```

Comment:
Because this JSX code does not use the `Card` component, this doesn't actually constitute a test for that component.